functions/go/apply-replacements/README.md

```diff
@@ -0,0 +1,107 @@
+# apply-replacements
+
+### Overview
+
+<!--mdtogo:Short-->
+
+Invoke the [kustomize replacements] feature as a KRM function. 
+
+<!--mdtogo-->
+
+### FunctionConfig
+
+<!--mdtogo:Long-->
+
+We use a `Replacements` object to configure the `apply-replacements` function. 
+
+```yaml
+apiVersion: kpt.dev/v1alpha1
+kind: Replacements
+metadata:
+  name: replacements-fn-config
+replacements:
+  # your replacements here
+```
+
+The syntax for `replacements` is described in the [kustomize replacements] docs. 
```

Comment:
suggest giving a simple example here and add the link to kustomize replacement.

Reply:
done!